rma_account/models/account_move.py

```diff
@@ -272,3 +269,40 @@ def _stock_account_get_anglo_saxon_price_unit(self):
                     self.company_id
                 )._compute_price(average_price_unit, self.product_uom_id)
         return price_unit
+
+    def _get_computed_name(self):
```

Comment:
Why is this method needed, that is basically what the compute method of the name field does right ?
https://github.com/odoo/odoo/blob/16.0/addons/account/models/account_move_line.py#L460

Reply:
Done!